subsystem/leader_test.go

```diff
@@ -227,15 +227,15 @@ var _ = Describe("Leader tests", func() {
 		log.Infof("Verify that previous leader was not changed")
 		waitForPredicate(timeout, test1.isLeader)
 	})
-	It("Deleting configmap in a loop", func() {
-		By("Deleting configmap in a loop (it must be recreated all the time), leader will loose leader and retake it")
+	It("Deleting lock underlying resource in a loop", func() {
```

Comment:
Should we also add a test for cleaning deprecated resources? If anything, just to test that we are not cleaning more than we should :)